site/management.md

```diff
@@ -401,24 +402,20 @@ In addition to the `connect-src` CSP header, RabbitMQ also needs the CSP directi
 
 By default, the RabbitMQ Management UI uses the OAuth 2.0 **authorization code flow** to authenticate and authorize users.
 However, there are scenarios where users preferred to be automatically redirected to RabbitMQ without getting
-involved in additional logon flows. This is common in Web Portals where with a single click, users navigate
-straight to a RabbitMQ cluster's management UI with a token obtained under the covers. This is known as
+involved in additional logon flows. This is common in Oauth2 proxies and in Web Portals where with a single click, users navigate straight to a RabbitMQ cluster's management UI with a token obtained under the covers. This is known as
 **Identity-Provider initiated logon**.
 
 RabbitMQ exposes a new setting called `management.oauth_initiated_logon_type` whose default value `sp_initiated`.
 To enable an **Identity-Provider initiated logon** you set it to `idp_initiated`.
 
-After you set `management.oauth_initiated_logon_type` to `idp_initiated` and
-`oauth_enabled: true` and `oauth_provider_url` are configured, the management UI exposes the HTTP endpoint `/login` which accepts `content-type: application/x-www-form-urlencoded` and it expects the JWT token in the `access_token` form field. This is the endpoint where the web portal would redirect users to access the RabbitMQ Management ui.
-
-This is the minimum required configuration for a RabbitMQ cluster configured with `idp_initiated` logon type:
-
 <pre class="lang-ini">
 management.oauth_enabled = true
 management.oauth_initiated_logon_type = idp_initiated
 management.oauth_provider_url = https://my-web-portal
 </pre>
 
+With the settings above, the management UI exposes the HTTP endpoint `/login` which accepts `content-type: application/x-www-form-urlencoded` and it expects the JWT token in the `access_token` form field. This is the endpoint where the web portal would redirect users to access the RabbitMQ Management ui.
```

Comment:
instead of the word "above" use "previous" try avoiding use words (where possible) like "above", "below", "top", and "bottom" etc in customer docs especially for readers with accessibility needs for example low visual users will not know where "above" is on a page.